Lucidtech/Lucidtech.csproj

```diff
@@ -15,7 +15,7 @@
     <TargetFrameworks>netstandard20</TargetFrameworks>
     <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
     <PackageId>Lucidtech.Las</PackageId>
-    <PackageVersion>1.2.0</PackageVersion>
+    <PackageVersion>1.2.1</PackageVersion>
```

Comment:
Should the version be 1.3.0? This update includes new non-breaking features, so I'd expect a jump in minor version